### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by IngeHeeringa**
Desktop view
image

Mobile view
image

### CODE REVIEW

File: components/Category.tsx
Comment (Gaston82) [inline components/Category.tsx:1]: We already have fonts as a variable .

File: app/produits/page.tsx
Comment (Gaston82) [inline app/produits/page.tsx:1]: We have a meta description in our root layout, I don't know if we should do it in every page. I'll have a look.

**Review by Gaston82 — APPROVED** [review]
Well done Miss Inge !!

File: components/Category.tsx
Comment (IngeHeeringa) [reply components/Category.tsx:1]: True, I'll change it

File: app/produits/page.tsx
Comment (IngeHeeringa) [reply app/produits/page.tsx:1]: I believe it's optionally possible to add page-specific metadata, but let's discuss how to approach this

**Pull request merged**